vyper/venom/venom_to_assembly.py

```diff
@@ -225,9 +234,23 @@ def _stack_reorder(
             if op == stack.peek(final_stack_depth):
                 continue
 
+            if len(positions[stack.peek(0)]) != 0:
+                positions[stack.peek(0)].remove(0)
+                insort(positions[stack.peek(0)], depth)
+
             cost += self.swap(assembly, stack, depth)
+            if final_stack_depth in positions[stack.peek(final_stack_depth)]:
```

Comment:
i think the `positions[stack.peek(final_stack_depth)]` can be factored out into a variable